src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/BlobCleanerTask.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.blobrouter.tasks;
+
+import org.slf4j.Logger;
+import uk.gov.hmcts.reform.blobrouter.config.ServiceConfiguration;
+import uk.gov.hmcts.reform.blobrouter.tasks.processors.ContainerCleaner;
+
+import java.util.stream.Stream;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+public class BlobCleanerTask {
```

Comment:
How about `DispatchedBlobCleanerTask`? It might be a bit more obvious, but not too important. Up to you.

Reply:
Later on we will possibly be cleaning not only dispatched blobs, I suggest to keep the class name as it is